### PR TITLE
Request: close warning c4018 on windows

### PR DESCRIPTION
**Pull request by lbfamous**
close warning c4018 on windows,too much warning on this

### CODE REVIEW

**Comment by lbfamous** [comment]
i don't know how to use github ..daomei

**Comment by minggo** [comment]
Thanks @lbfamous , but i can not merge this pull request because there is Chinese words in commit log and in source code.

Could you please use English and re-send a pull request again?

File: cocos/platform/CCPlatformConfig.h [hunk 1]
```diff
@@ -154,6 +154,7 @@ THE SOFTWARE.
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_WIN32)
 #ifndef __MINGW32__
 #pragma warning (disable:4127) 
+#pragma warning (disable:4018)
```
Comment (WenhaiLin) [inline cocos/platform/CCPlatformConfig.h:157]: warnings should be fixed, not hidden..

Comment (lbfamous) [inline cocos/platform/CCPlatformConfig.h:157]: too many files contanins this warning..difficult on close all that.a easy  is disable this warning

Comment (lbfamous) [inline cocos/platform/CCPlatformConfig.h:157]: I don't know how to delete the commit log? can somebody help me??

**Pull request closed without merging**